packages/duration/src/lib/constants.ts

```diff
@@ -1,13 +1,16 @@
 import type { DurationFormatAssetsTime, DurationFormatSeparators } from './DurationFormatter';
 
 export enum Time {
+	Nanosecond = 1 / 1_000_000,
+	Microsecond = 1 / 1000,
 	Millisecond = 1,
 	Second = 1000,
-	Minute = 1000 * 60,
-	Hour = 1000 * 60 * 60,
-	Day = 1000 * 60 * 60 * 24,
-	Month = 1000 * 60 * 60 * 24 * (365 / 12),
-	Year = 1000 * 60 * 60 * 24 * 365
+	Minute = Second * 60,
+	Hour = Minute * 60,
+	Day = Hour * 24,
+	Week = Day * 7,
+	Month = Day * (365 / 12),
```

Comment:
Will I be pedantic about the fact a year isn't exactly 365 days, and a month isn't technically `365/12`? yes. 🤣

Reply:
Tbh we had that in the past too so lets not be pedantic lol

Reply:
I can do a semver-major change to make them `365.25` (and also normalize it everywhere), wdyt?

Reply:
I don't really want a semver major for that tbh

Reply:
What, it wouldn't be semver major to change it, I'd consider this a patch 🤔 . Can you explain why this would be major? 👁️

Reply:
That still doesn't answer my question! Yes, people rely on these values, but they also just use the enum access, and not the raw number 🤔 
It also only affects the biggest units, which most people don't use anyways, and I'd rather release a patch that have the *correct* values rather than avoid fixing this because its pseudo-major

Reply:
well.. fine.. so what will it be then? Either way, @kyranet can you fix that in #562 and change the commits from `refactor` to `fix`? This time you *are* going to have to rebase and use `edit` on each Kyra, sorry.